modules/howtos/examples/Analytics.java

```diff
@@ -86,7 +86,7 @@ public static void main(String... args) {
     {
       // tag::named[]
       AnalyticsResult result = cluster.analyticsQuery(
-        "select count(*) from airports where country = $country",
+        "select count(*) from `travel-sample`.inventory.airport where country = $country",
```

Comment:
So, this file *already* has Collection examples in it, that I did in DOC-7628.
As @dnault suggests, it might be better to keep some of the non-Collection examples.

Gonna build it now and have a look at it in context

Reply:
Yeah, as discussed on Slack, this isn't quite right - as the collections examples at the *end* are worded as if they're introducing the concept, despite it being used throughout.

Perhaps that section should be moved to top, or we should revert to Default collection examples in this case

Reply:
Yeah, I think for the pages where we already have a section for named scopes and collections I will revert the examples to default as it probably makes more sense there.